CMakeLists.txt

```diff
@@ -1,91 +1,227 @@
-# 
+# ~~~
 # Copyright(c) 2019 Intel Corporation
```

Comment:
why add ~~~

Reply:
cmake-format tries to format the copy right thinking that it's a normal comment.

Might uninstall cmake-format due to these kinds of issues